src/hooks/useAppInsights.js

```diff
@@ -0,0 +1,66 @@
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+
+export default function useAppInsights() {
+  const ai =
+    typeof window !== 'undefined' &&
+    process.env.APPLICATIONINSIGHTS_CONNECTION_STRING &&
+    new ApplicationInsights({
+      config: {
+        connectionString: process.env.APPLICATIONINSIGHTS_CONNECTION_STRING,
+        extensionConfig: {
+          ['AppInsightsCfgSyncPlugin']: {
+            cfgUrl: '',
+          },
```

Comment:
This isn't very obvious what it does... Can you add a comment with a link to the PBI?

Reply:
✅ done